packages/sdk/src/error.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  AuctionResponse,
+  CancelParams,
+  FulfillParams,
+  jsonifyError,
+  NxtpError,
+  PrepareParams,
+} from "@connext/nxtp-utils";
+import { providers } from "ethers";
+
+/**
+ * @classdesc Abstract error class thrown by the `TransactionManager` class.
+ */
+export class TransactionManagerError extends NxtpError {
+  static readonly type = TransactionManagerError.name;
+}
+
+/**
+ * @classdesc Error thrown by the `TransactionManager` class if no transactionManager was available for the specified chain.
+ */
+export class ChainNotSupported extends TransactionManagerError {
+  static readonly type = ChainNotSupported.name;
+
+  constructor(
+    public readonly method: string,
+    public readonly methodId: string,
+    public readonly chainId: number,
+    public readonly context: {
+      transactionId?: string;
+      approveReceipt?: providers.TransactionReceipt;
+      assetId?: string;
+      amount?: string;
+      router?: string;
```

Comment:
Not sure if this is relevant to the error condition you're addressing here

Reply:
Me neither, I just added args bc I would rather err on side of too detailed

Reply:
should I remove amount/router? assetId remove as well?